CMakeLists.txt

```diff
@@ -144,6 +133,7 @@ message(STATUS "Setting Namespace to: ${PROJ_NAMESPACE_V}")
 add_definitions (-DOIIO_INTERNAL=1)
 
 list (APPEND CMAKE_MODULE_PATH
+      "${PROJECT_SOURCE_DIR}/build"
```

Comment:
Why? There aren't going to be any cmake modules in this location, which won't yet exist when this cmake file is processed and the modules are needed.

Reply:
Conan will create cmake files there.

Reply:
But `${PROJECT_SOURCE_DIR}/build` is the wrong place. If they invoke cmake (`-B`, for example) with any other directory, it could be any other location. I think you want `${CMAKE_BINARY_DIR}` at the very least.

Reply:
"Fixed" in 803b05c05cae45f79d6f809dff2aeb52ac846231